src/main/java/apoc/schema/Schemas.java

```diff
@@ -117,6 +135,101 @@ private AssertSchemaResult createUniqueConstraint(Schema schema, String lbl, Str
         return new AssertSchemaResult(lbl, key).unique().created();
     }
 
+    public List<AssertSchemaResult> recreateIndexes(long allIndexesTimeout, long dropAllIndexesTimeout, long recreateAllIndexesTimeout) throws ExecutionException, InterruptedException, IllegalArgumentException, TimeoutException {
+        Schema schema = db.schema();
+        List<AssertSchemaResult> result = Collections.synchronizedList(new ArrayList<>());
+        ExecutorService executorService = Pools.SINGLE;
```

Comment:
why not Pools.DEFAULT?

Reply:
Wanted it to be single threaded. As it is it woudn't make much difference as do blocking futures anyway

Reply:
ok